src/Angor/Shared/Models/Outpoint.cs

```diff
@@ -17,4 +17,14 @@ public OutPoint ToOutPoint()
     {
         return new OutPoint(uint256.Parse(transactionId), outputIndex);
     }
+
+    public static Outpoint FromOutPoint(OutPoint outPoint)
+    {
+        return new Outpoint { outputIndex = (int)outPoint.N, transactionId = outPoint.Hash.ToString() };
+    }
+
+    public static Outpoint Create(string trxid, int index)
```

Comment:
Shouldn't this just be a constructor?

Reply:
I think its simpler to understand as it is used in linq queries